src/components/Article.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+const Article = ({ title, date, text }) => (
+  <div>
+    <h1>{title}</h1>
+    <span>{date}</span>
+    <p>{text}</p>
+  </div>
+);
+
+Article.defaultProps = {
+  title: '',
+  date: '',
+  text: '',
+};
```

Comment:
It doesn't really make sense to render empty article if no data is received. It's better to make these fields required.